hal-base/hal/functions.py

```diff
@@ -494,14 +484,14 @@ def cleanNotifier(notifier):
 #############################################################################
 # PDP
 #############################################################################
-getPDPTemperature = _STATUSFUNC("getPDPTemperature", C.c_double)
```

Comment:
See the [halfix](https://github.com/robotpy/allwpilib/blob/halfix/hal/include/HAL/PDP.hpp) branch in allwpilib.